### PR TITLE
skip tests on documentation only edits

### PR DESCRIPTION
**Pull request by jbolda**
This should "scope" test running to when there is a real value in running them, ie when the code changes. If the documentation or website is the only change, there isn't a real need to run the tests.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -44,7 +47,21 @@ jobs:
...
+            logging,
+            bundler,
+            project,
+            server,
+            suite,
+            interactor,
+            todomvc,
+            atom,
+            webdriver,
+          ]
```
Comment (cowboyd) [inline .github/workflows/test.yml]: is this related?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -92,4 +109,4 @@ jobs:
         with:
           working-directory: packages/interactor/
           start: yarn bigtest-todomvc 3000
-          wait-on: 'http://localhost:3000' 
\ No newline at end of file
+          wait-on: "http://localhost:3000"
```
Comment (cowboyd) [inline .github/workflows/test.yml]: seems like a formatting change.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -3,7 +3,10 @@ name: Test
 on:
   pull_request:
     branches:
-      - 'v[0-9]+'
+      - "v[0-9]+"
+    paths-ignore:
+      - "website/**"
+      - "**.md"
```
Comment (cowboyd) [inline .github/workflows/test.yml]: Will this match the readmes in the individual packages? If so, we should avoid those since that will actually change the content of what gets published to npm.

**Review by cowboyd — CHANGES_REQUESTED** [review]
YAASSSSSSSSSS!!!! 

want!

**Review by cowboyd — CHANGES_REQUESTED** [review]
YASSSS! WANT!

File: .github/workflows/test.yml (hunk 2 above)
Comment (jbolda) [reply .github/workflows/test.yml]: ah, blast, I reconfigured things (in my personal setup) and must have left auto-formatting of yml on. I'll revert this and the package list change.

File: .github/workflows/test.yml (hunk 3 above)
Comment (jbolda) [reply .github/workflows/test.yml]: Yes it will match those. True that it will change what gets published, but running the tests for a readme change isn't at all connected to publishing, is it?

Comment (cowboyd) [reply .github/workflows/test.yml]: That would be weird wouldn't it!

**Review by cowboyd — APPROVED** [review]
